### PR TITLE
web-ext: add page

### PR DESCRIPTION
**Pull request by pxgamer**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform folder (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/web-ext.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# web-ext
+
+> A command line tool for managing web extension development.
+
```
[the remark this answers is not in the document]
Comment (pxgamer) [reply pages/common/web-ext.md]: I wasn't sure whether it's better to use the npm link, or their GitHub repository:
https://github.com/mozilla/web-ext

npm sounds good to me. 👍

File: pages/common/web-ext.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# web-ext
+
+> A command line tool for managing web extension development.
+> Homepage: <https://www.npmjs.com/package/web-ext>.
+
+- Open the current web extension in Firefox:
```
Comment (mebeim) [inline pages/common/web-ext.md]:
```suggestion
- Run the web extension in the current directory in Firefox:
```

File: pages/common/web-ext.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`web-ext run`
+
+- Open a specific directory's web extension in Firefox:
```
Comment (mebeim) [inline pages/common/web-ext.md]:
```suggestion
- Run a web extension from a specific directory in Firefox:
```

File: pages/common/web-ext.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`web-ext run --source-dir {{path/to/directory}}`
+
+- Display verbose execution output:
+
+`web-ext run --verbose`
+
+- Open a web extension in Firefox Android:
```
Comment (mebeim) [inline pages/common/web-ext.md]:
```suggestion
- Run a web extension in Firefox Android:
```

File: pages/common/web-ext.md (hunk 3 above)
Comment (pxgamer) [reply pages/common/web-ext.md]: This and the previous comment are literally what I had when I originally wrote the page. :yum: I'll apply these changes in the morning. :+1: Thanks.

**Pull request merged**